chain/chain/src/stateless_validation/chunk_validation.rs

```diff
@@ -67,6 +67,9 @@ impl MainTransition {
 pub struct PreValidationOutput {
     pub main_transition_params: MainTransition,
     pub implicit_transition_params: Vec<ImplicitTransitionParams>,
+    /// List of the transactions that are valid and should be validated. False transactions get
```

Comment:
did you mean "and should be **executed**"?

Reply:
I guess. I was thinking of "validated" as in "validation" step of the whole protocol thing, but I guess that makes the sentence pretty odd to read, doesn't it?